src/components/ButtonStyled/index.js

```diff
@@ -0,0 +1,55 @@
+import { Button, useTheme } from '@octopusthink/nautilus';
+import React from 'react';
+import { css } from '@emotion/core';
+
+const ButtonStyled = (props) => {
```

Comment:
"ButtonStyled" is a bit of an odd component name, maybe `TurnipButton` or something? I dunno, even just `Button` seems better 🤷🏻‍♂️

Reply:
You fixed this already, but noted on the approach—makes more sense! 👍